ci-operator/step-registry/hypershift/azure/create/hypershift-azure-create-chain.yaml

```diff
@@ -18,7 +18,12 @@ chain:
       documentation: "The Azure location of the cluster."
     - name: ENABLE_ICSP
       default: "false"
-      documentation: "if true, add image content sources config(path=${SHARED_DIR}/mgmt_icsp.yaml)"
+      documentation: "If true, add image content sources config(path=${SHARED_DIR}/mgmt_icsp.yaml)"
+    - name: HYPERSHIFT_ETCD_ENCRYPTION
+      default: "false"
+      documentation: |-
+        If true, enable ETCD encryption by specifying 
+        --encryption-key-id="$(cat "${SHARED_DIR}/hypershift_azure_etcd_encryption_active_key_url")".
```

Comment:
This key is only used to encrypt etcd ?

Actually installer also some encryption coverage using vault key pairs, e.g: `azure-provision-disk-encryption-set`, we are using it to encrypt OS disk.

And there is also an existing etcd-encryption step -`etcd-encryption`.

Reply:
Yup, it only used for ETCD encryption. There's another flag for Azure disk encryption. The `etcd-encryption` step is not applicable for Hypershift ETCD encryption. Anyways, thanks for the information.